modules/authorization/policy-definitions/management-group/metadata.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://aka.ms/bicep-registry-module-metadata-file-schema-v2#",
+  "name": "Policy Definitions on Management Group level",
+  "summary": "With this module you can create policy definitions on a management group level.",
```

Comment:
Should the README.md be updated to "This module deploys Policy Definitions on a Management Group level." and then this file updated accordingly?